src/scripts/ordo_subsets.py

```diff
@@ -0,0 +1,84 @@
+"""Creates a ROBOT template to update mondo terms with the correct ORDO subset annotation."""
+from argparse import ArgumentParser
+
+import pandas as pd
+
+from utils import remove_angle_brackets
+
+
+SUBSET_MAP = {
+    'Orphanet:557493': 'http://purl.obolibrary.org/obo/mondo#ordo_disease',  # disorder
```

Comment:
```suggestion
    'Orphanet:557493': 'http://purl.obolibrary.org/obo/mondo#ordo_disorder',  # disorder
```

Wait for discussion in `#mondo-ingest` slack channel to conclude.

Reply:
I take it that is [this Slack convo](https://monarchinitiative.slack.com/archives/C03AW548J8M/p1716486163402139).